torch/_inductor/codegen/triton.py

```diff
@@ -1068,7 +1068,11 @@ def gen_assert_indirect_indexing(self, buffer, index, mask):
                     indirect_bounds[indirect_name[indirect_var]] = rg
 
         def gen_min_max(var, size):
-            if dynamo_config.dynamic_shapes:
+            # TODO At the moment we just give per-LoopBody bounds. This is why
+            # we may have a `var not in indirect_bounds`. We should be able
+            # to perform this analysis for a whole SchedulerNode! Same for
+            # any other analysis that uses BoundVars
+            if dynamo_config.dynamic_shapes or var not in indirect_bounds:
```

Comment:
was this the case prior to the refactoring ?

Reply:
yeah, the implementaiton just deals with one `LoopBody` at a time.

Reply:
I looked into doing it across LoopBodies but all of the cases I was looking at that didn't actually help, for what it's worth